torch/nn/_functions/rnn.py

```diff
@@ -377,11 +363,9 @@ def forward(input, *fargs, **fkwargs):
 
         # Hack for the tracer that allows us to represent RNNs as single
         # nodes and export them to ONNX in this form
-        if torch._C._jit_is_tracing(input):
-            assert not fkwargs
-            output = func(input, *fargs)
-            return hack_onnx_rnn((input,) + fargs, output, args, kwargs)
-        else:
-            return func(input, *fargs, **fkwargs)
+        # It can be also used as a decorator at the higher level
+        func = RNN_symbolic_builder(*args, **kwargs)(func)
```

Comment:
This part is very confusing. Both the outer function and its insides are decorated. I think only one of those would be sufficient (and any would do the job).